src/main/java/com/rusiecki/jesttest/service/CustomService.java

```diff
@@ -0,0 +1,88 @@
+package com.rusiecki.jesttest.service;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.rusiecki.jesttest.model.Article;
+import com.rusiecki.jesttest.model.BaseDto;
+import com.rusiecki.jesttest.model.Post;
+import io.searchbox.client.JestClient;
+import io.searchbox.client.JestResult;
+import io.searchbox.core.Search;
+import org.elasticsearch.index.query.QueryBuilders;
+import org.elasticsearch.search.builder.SearchSourceBuilder;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+@Component
+public class CustomService {
+    @Autowired
+    private JestClient client;
+
+    public List findAll() {
+        SearchSourceBuilder searchSourceBuilder = new SearchSourceBuilder();
+        searchSourceBuilder.query(QueryBuilders.matchAllQuery());
+        Search search = new Search.Builder(searchSourceBuilder.toString())
+                .addIndex(ArticleService.INDEX)
+                .addIndex(PostService.INDEX)
+                .addType(SimpleCrudService.TYPE)
+                .build();
+        JestResult result;
+        try {
+            result = client.execute(search);
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
Log the exception and throw your own custom runtime exception, than handle it in the global exception handler